src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.scss

```diff
@@ -1,3 +1,3 @@
 .bcs-TaskCompletionRuleIcon {
-    margin-left: 4px;
+    margin-left: 5px;
```

Comment:
This should be on the 4px grid. What was off here (let's include some screenshots)